README.md

```diff
@@ -2,55 +2,48 @@
 
 ## About
 
-The ROOT system provides a set of modules with all the functionality
-needed to handle and analyze large amounts of data in a very efficient way.
-Having the data defined as a set of objects, specialized storage methods are
-used to get direct access to the separate attributes of the selected objects,
-without having to touch the bulk of the data. Included are histograming
-methods in an arbitrary number of dimensions, curve fitting, function
-evaluation, minimization, graphics and visualization classes to allow
-the easy setup of an analysis system that can query and process the data
+ROOT is a unified software package for the storage, processing, visualization 
+and analysis of scientific data. It is reliable, performant and well supported,
+easy to use and obtain, and strives to minimize the computing resources needed 
+to achieve scientific results.
+
+ROOT is a comprehensive toolbox to process and analyze data, from its acquisition, 
+to its processing, analysis and final visualization in form of highly customizable, 
+publication-ready plots.
```

Comment:
Does that not repeat the previous paragraph?

Reply:
In a sense yes.